vision/lidar/record_data.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+""" Record data from Ouster OS-1-32-U LiDAR sensor to pcap file."""
+
+import os
+import sys
+import argparse
+import time
+from contextlib import closing
+from datetime import datetime
+from more_itertools import time_limited
+
+from ouster import client, pcap
+
+HOSTNAME = "os-122229001687.local"
+LIDAR_PORT = 7502
+IMU_PORT = 7503
+
+
+def record_pcap(
+    hostname: str,
+    lidar_port: int = 7502,
+    imu_port: int = 7503,
+    fname_base: str | None = None,
+    n_seconds: int = 10,
+) -> None:
+    """Record data from live sensor to pcap file.
+
+    Note that pcap files recorded this way only preserve the UDP data stream and
+    not networking information, unlike capturing packets directly from a network
+    interface with tools like tcpdump or wireshark.
```

Comment:
Formatting so it's colored in VS Code
```suggestion
    NOTE: pcap files recorded this way only preserve the UDP data stream and
    not networking information, unlike capturing packets directly from a network
    interface with tools like tcpdump or wireshark.
```